### ISSUE TITLE
start/stop points falling on rows are ignored

### ISSUE THREAD

**Issue by lexelby** [inkstitch/inkstitch#313]
If you set the start or end point on the top or bottom side of a fill region, Ink/Stitch snaps the point to the left or right side of that row (whichever is closer).

**Comment by ndrwclrk** [comment]
Not completely sure, but I may have discovered either a refinement of, or a workaround for this issue.  I have an object for which I'm trying to set the start and stop points manually.  Ink/Stitch seemed to be ignoring the manually set connector points.  However, I found I could get the result I want in the following way:
1. Place the symbol on the workspace and use the Flow Chart connectors to find out where the centres of the objects are. 
2. Move the symbol so that the path from the centre of the fill object to the symbol crosses the border of the fill object exactly where you want the start/end point. 
In other words, if you carefully place the symbol so that the default routing of the connector from centre to centre just happens to fall on the point you want, it seems to work.

I've tested it in a few objects, and so far this works.  It might fail on some objects though. I'm not sufficiently familiar with the code to analyze why this might work, and when it might fail.

**Comment by lexelby** [comment]
> In other words, if you carefully place the symbol so that the default routing of the connector from centre to centre just happens to fall on the point you want, it seems to work.

Ah, yes.  Here's what's going on there: Ink/Stitch will pay attention to where you put the connector endpoint if you edit it manually.  However, Inkscape really wants to put it in its "natural" place, that is, on the border of the object, along a line between the centers of the two objects.  You can set it manually all you like, but any time you run any extension, Inkscape will move all the connectors for you.

It's pretty annoying, but I decided to stick with it because in all other ways the flowchart connector system is a pretty good fit for attaching commands to objects.  Sorry it's a bit of a pain :/

That's not actually the issue that I was talking about in my original post here.  I think you might be seeing a combination of the two problems, which probably makes it confusing.

For my original post, imagine you're filling a rectangular shape:

```
 _______
|       |
|       |
|       |
|       |
|_______|
```

Ink/Stitch will generate a bunch of parallel lines that fit in the shape:
```
 _______
|_______|
|_______|
|_______|
|_______|
|_______|
```

If you point your start or end position along the outside border at the top or bottom:

```
 _______
|_______|
|_______|
|_______|
|_______|
|_______|
    \
     \
      \
       >
```

Then Ink/Stitch will choose the nearest endpoint of one of those parallel lines:

```
 _______
|_______|
|_______|
|_______|
|_______|
|_______|
        \
         \
          \
           >
```

That's not what we want; we want it to obey the actual spot we chose.

**Comment by kaalleen** [comment]
@lexelby is this still an issue?

**Comment by lexelby** [comment]
Nope! Fixed that one.
